src/atomate2/classical_md/openmm/jobs/base.py

```diff
@@ -0,0 +1,494 @@
+        To record no state, set to 0.
+    state_file_name : Optional[str]
+        The name of the state file to save.
+    traj_interval : Optional[int]
+        The interval for saving trajectory frames. To record
+        no trajectory, set to 0.
+    wrap_traj : Optional[bool]
+        Whether to wrap trajectory coordinates.
+    report_velocities : Optional[bool]
+        Whether to report velocities in the trajectory file.
+    traj_file_name : Optional[str]
+        The name of the trajectory file to save.
+    traj_file_type : Optional[str]
+        The type of trajectory file to save. Supports any output format
+        supported by MDAnalysis.
+    """
+
+    name: str = "base openmm job"
+    tags: list[str] | None = field(default=None)
+    n_steps: int | None = field(default=None)
+    step_size: float | None = field(default=None)
+    temperature: float | None = field(default=None)
+    friction_coefficient: float | None = field(default=None)
+    platform_name: str | None = field(default=None)
+    platform_properties: dict | None = field(default=None)
+    state_interval: int | None = field(default=None)
+    state_file_name: str | None = field(default=None)
+    traj_interval: int | None = field(default=None)
+    wrap_traj: bool | None = field(default=None)
+    report_velocities: bool | None = field(default=None)
+    traj_file_name: str | None = field(default=None)
+    traj_file_type: str | None = field(default=None)
+    embed_traj: bool | None = field(default=None)
+
+    @openff_job
+    def make(
+        self,
+        interchange: Interchange | bytes,
+        prev_task: OpenMMTaskDocument | None = None,
+        output_dir: str | Path | None = None,
+    ) -> Response:
+        """Run an OpenMM calculation.
+
+        This method sets up an OpenMM simulation, runs the simulation based
+        on the specific job logic defined in run_openmm, and closes the
+        simulation. It returns a response containing the output task document.
+
+        Parameters
+        ----------
+        interchange : Union[Interchange, str]
+            An Interchange object or a JSON string of the Interchange object.
+        prev_task : Optional[OpenMMTaskDocument]
+            The previous task document.
+        output_dir : Optional[Union[str, Path]]
+            The directory to save the output files.
+            Resolution order is output_dir > prev_task.dir_name > Path.cwd(). Will
+            create a directory if needed.
+
+        Returns
+        -------
+        Response
+            A response object containing the output task document.
+        """
+        # this is needed because interchange is currently using pydantic.v1
+        if isinstance(interchange, bytes):
+            interchange = interchange.decode("utf-8")
+        if isinstance(interchange, str):
+            interchange = Interchange.parse_raw(interchange)
+        else:
+            interchange = copy.deepcopy(interchange)
+
+        dir_name = Path(
```

Comment:
I wouldn't expect to create a new directory? This will already be handled by the workflow manager (e.g., jobflow, jobflow-remote, or fireworks). I think we should remove this to be consistent with the other workflows.